site/content/docs/5.2/forms/form-control.md

```diff
@@ -114,9 +114,20 @@ If you want to have `<input readonly>` elements in your form styled as plain tex
 
 Set the `type="color"` and add `.form-control-color` to the `<input>`. We use the modifier class to set fixed `height`s and override some inconsistencies between browsers.
 
+{{< callout info >}}
+{{< partial "callouts/warning-color-picker.md" >}}
```

Comment:
IMO I wouldn't use a partial if only used at one location. Moreover, the level of heading could change if it became reusable at several locations.